Make things a bit easier ot understand/absorb (#1)
QuiltServerTools/ServerSideDevDocs: docs/libraries.md

}
```

> You now add a maven repository with the URL that links to it:

Reviewer: ```suggestion
You now need to add the maven repository's URL to the section:
```

Author: Yeah I was most unsure on how to word that because it made little sense to me in the first place :P
